core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala

```diff
@@ -205,13 +207,15 @@ class WorkflowSubmissionSpec(_system: ActorSystem) extends TestKit(_system) with
       }
 
       val token = Await.result(workflowSubmission.googleServicesDAO.getToken(testData.submission1.submitter), Duration.Inf).get
+      val petJson = Await.result(workflowSubmission.samDAO.getPetServiceAccountKeyForUser(testData.workspace.namespace, testData.userOwner.userEmail), Duration.Inf)
       assertResult(
         Some(
           ExecutionServiceWorkflowOptions(
             s"gs://${testData.workspace.bucketName}/${testData.submission1.submissionId}",
             testData.wsName.namespace,
             testData.userOwner.userEmail.value,
-            token,
+            "pet-110347448408766049948@broad-dsde-dev.iam.gserviceaccount.com",
```

Comment:
Is this a real pet or just an example? If real, there should be some indication of who its tied to. If just an example that doesn't need to really exist, it would help if that was obvious at a glance.

I just want to be able to see some link that proves that the test is in control of the data its asserting.

Reply:
I just copied a real json payload